src/web/components/RightColumn.tsx

```diff
@@ -10,18 +10,22 @@ const hideBelowDesktop = css`
 
     ${from.desktop} {
         /* above 980 */
+        height: 100%;
         display: block;
         flex-basis: 300px;
         flex-grow: 0;
         flex-shrink: 0;
     }
 `;
 
+const hideForIE = css`
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```

Comment:
Tested with browserstack using IE11 on Windows 10

Reply:
+1

Reply:
To be clear, if we're hiding the right column for IE, does this mean the ads that normally appear in the right column at desktop and above will not appear at all for IE11 users?

Reply:
Only the top most advert will no longer appear. The other in body ads will still be rendered to the right.

If we wanted to restore that top most right column advert we could patch something to absolute position that element for IE.

Reply:
As discusses irl, we'll send out comms for this change and based on feedback decide if we need to restore this ad slot later